### ISSUE TITLE
Storybook broken

### ISSUE THREAD

**Issue by gijs** [nens/G4AW-lizard-client#81]
PR #77 breaks React Storybook.

It [adds the `.css` extension to the webpack extensions array](https://github.com/nens/G4AW-lizard-client/pull/77/commits/bea8c3f7e14aebbe1ae0564dd9f3b8c4ed4d55f8#diff-11e9f7f953edc64ba14b0cc350ae7b9d).

This works fine when running `yarn start` but not when running `yarn run storybook`, because the storybook looks at `.storybook/webpack.config.js`.
The `extensions` array is not overridable according to the Storybook documentation. (tried it - failed)

In the file `node_modules/@kadira/storybook/dist/server/webpack.config.js`, there is an array `extensions` on line 32. Adding `.css` to it fixes the problem, but is not a real solution. 

TL;DR: To make it work quickly, make L32 in `node_modules/@kadira/storybook/dist/server/webpack.config.js` look like this `extensions: ['.js', '.json', '.jsx', '.css', ''],`

**Comment by gijs** [comment]
@d-weeteling Maybe the easiest solution it to try upgrading Storybook or else revert your change to resolve the `.css` extension.

**Comment by gijs** [comment]
Fixed in master for now
